src/PizzaForm.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import * as yup from 'yup';
+import axios from 'axios';
+
+function PizzaForm() {
+const [pizzaForm, setPizzaForm] = useState({
+    name: '',
+    size: 'small',
+    pepperoni: false,
+    pineapple: false,
+    anchovies: false,
+    ham: false,
+    sausage: false,
+    instructions: ''
+})
+const [disabled, setDisabled] = useState(true);
+const [order, setOrder] = useState();
+const [serverError, setServerError] = useState([]);
+const [inlineError, setInlineError] = useState({
+    name: ''
+});
+const handleNameChange = e => {
+    e.persist();
+    validate(e);
+    setPizzaForm({...PizzaForm, [e.target.name]: e.target.value})
```

Comment:
I think you have a typo with `PizzaForm`, which should be `pizzaForm`
You may want to consider merging `handleNameChange` and `handleChange` to keep your code DRY. You could have a conditional to determine whether or not to do the validation. If this were one function then maybe you wouldn't have had this typo :)